SwiftBuddiesIOS/Sources/TabFlowView.swift

```diff
@@ -1,55 +1,53 @@
 import SwiftUI
-import Auth
-import Map
 import Feed
-import Onboarding
-import About
+import Map
+import Profile
 import Contributors
-import Design
 
-public struct ContentView: View {
-    @AppStorage("isSplashScreenViewed") var isOnboardingScreenViewed : Bool = false
-    
-    public init() { }
-
-    public var body: some View {
-        SuitableRootView()
-    }
+enum AppTab: Int, Identifiable {
+    case feed = 0
+    case map
+    case profile
+    case contributors
     
-    @ViewBuilder
-    private func SuitableRootView() -> some View {
-        if isOnboardingScreenViewed {
-            TabFlow()
-        } else {
-            OnboardingBuilder.build()
-        }
-    }
+    var id: Int { rawValue }
 }
 
-struct TabFlow: View {
+struct TabFlowView: View {
+    
     @State var selectedTab: AppTab = .feed
-
-    public init() {}
+    @Binding private var showSignInView: Bool
+    
+    init(showSignInView: Binding<Bool>) {
+        self._showSignInView = showSignInView
+    }
+    
     var body: some View {
         TabView(selection: $selectedTab) {
+            
             FeedView()
                 .tabItem {
                     Image(systemName: "list.bullet")
                     Text("Feed")
                 }
                 .tag(AppTab.feed)
+            
             MapView()
                 .tabItem {
                     Image(systemName: "map")
                     Text("Map")
                 }
                 .tag(AppTab.map)
-            AboutView()
-                .tabItem {
-                    Image(systemName: "info.circle")
-                    Text("About")
-                }
-                .tag(AppTab.about)
+            
+            NavigationStack {
+                ProfileView(showSignInView: $showSignInView)
```

Comment:
Bu binding sadece sign in view'unu present etmek icin kullaniliyorsa boyle yapmaktansa NotificationCenter ile notification yollanabilir. Ki eninde sonunda boyle bir yapi baska ekranlar icin gerekebilir o yuzden `showSignInView` binding'ine gerek yok